lms/templates/header/header.html

```diff
@@ -29,6 +32,13 @@
 % endif
 
 <header class="global-header ${'slim' if course else ''}">
+    % if use_cookie_banner and ENABLE_GDPR_COMPAT_FLAG.is_enabled_without_course_context():
+        ${static.renderReact(
+            component="CookiePolicyBanner",
+            id="cookie-policy-banner",
```

Comment:
Where is this element on the page? I was assuming you would use the already present ```#page-prompt``` wrapper?

Reply:
![image](https://user-images.githubusercontent.com/8136030/38685704-8210a35c-3e40-11e8-8024-7125b837ebb6.png)

Reply:
Is `#page-prompt` used to display other alerts across the site? Because this might override those alerts?